studio/__main_unit__.py

```diff
@@ -92,6 +92,11 @@ async def startup_event():
     logging.info('"Studio" application startup complete.')
```

Comment:
@quanpython 
Please keep a log of the operating mode here.

- > "Studio" application startup complete. [mode: (standalne OR multiuser)]